### PR TITLE
Make whisper nodes reachable from prometheus by sharing a network

### PR DESCRIPTION
**Pull request by dshulyak**
Tested, nodes are up and scrapable.

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]
Looks good. Was it tested on Linux or macOS or both?

**Comment by dshulyak** [comment]
Only on linux, I think there will be no difference with macos but would be good to make a run.

**Comment by dshulyak** [comment]
@adambabik regarding dynamic discovery, this https://github.com/prometheus/prometheus/pull/3687 seems like the best idea. If it won't be merged in next 1-2 weeks we can rebuild container with this change on top of stable prometheus release.

**Comment by mandrigin** [comment]
@dshulyak can you provide a very simple "how to test" instructions so I can check on the macOS? It shouldn't be a problem, but networking in Docker for Mac is quite different from Linux.

**Comment by dshulyak** [comment]
@mandrigin from whisper directory `docker-compose up -d --scale wnode=3`
then prometheus `docker-compose up -d`
prometheus will be available on localhost:9090, wait a bit till the nodes will be registered

**Comment by mandrigin** [comment]
@dshulyak I don't see anything in Prometheus except all the nodes showing as *UP*. Probably that is a good sign :)

File: cmd/statusd/main.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ var (
 
 	// stats
 	statsEnabled = flag.Bool("stats", false, "Expose node stats via /debug/vars expvar endpoint or Prometheus (log by default)")
-	statsAddr    = flag.String("stats.addr", "127.0.0.1:8080", "HTTP address with /debug/vars endpoint")
+	statsAddr    = flag.String("stats.addr", "0.0.0.0:8080", "HTTP address with /debug/vars endpoint")
```
Comment (adambabik) [inline cmd/statusd/main.go:51]: Another option would be to leave `127.0.0.1` and set it in docker-compose in `wnode-test-cluster/` with `-stats-addr=0.0.0.0:8080` but I think this is fine as well.

**Review by adambabik — APPROVED** [review]
Nice!

**Comment by adambabik** [comment]
@dshulyak swarm discovery would be very nice. We can wait a bit. There is also a way [to set up discovery using a JSON file](https://www.robustperception.io/using-json-file-service-discovery-with-prometheus/). It's pretty generic but it requires an additional script. Also, it will be even more work if would like to make it dynamic I guess.

@mandrigin yeah, it's a good sign :D

**Pull request merged**